TOC-tidb-cloud-essential.md

```diff
@@ -214,7 +214,8 @@
 - Migrate or Import Data
   - [Overview](/tidb-cloud/tidb-cloud-migration-overview.md)
   - Migrate Data into TiDB Cloud
-    - [Migrate and Merge MySQL Shards of Large Datasets](/tidb-cloud/migrate-sql-shards.md)
+    - [Migrate from TiDB Self-Managed to TiDB Cloud](/tidb-cloud/migrate-from-op-tidb.md)
+    - [Migrate from MySQL-Compatible Databases Using AWS DMS](/tidb-cloud/migrate-from-mysql-using-aws-dms.md)
```

Comment:
```suggestion
```

Reply:
At a closer look, I just find that "Migrate from MySQL-Compatible Databases Using AWS DMS" applies only to TiDB Cloud Dedicated 

<img width="1384" height="1436" alt="image" src="https://github.com/user-attachments/assets/60c2cc3a-b4fa-4296-9c4a-bb396e3a73fa" />